content/applications/hr/recruitment.rst

```diff
@@ -0,0 +1,726 @@
+- :guilabel:`Location`: Enter the location for the meeting.
+- :guilabel:`Meeting URL`: If the meeting is virtual, enter the meeting link in this field.
+- :guilabel:`Tags`: Select any tag(s) for the meeting using the drop-down menu. There is no limit to
+  the number of tags that can be used.
+
+Options tab
+^^^^^^^^^^^
+
+- :guilabel:`Recurrent`: If the meeting should repeat at a selected interval (not typical for a
+  first interview), check the box next to :guilabel:`Recurrent`. This reveals a recurrence module.
+  Enter the details for how often the meeting should repeat, including an end time.
+- :guilabel:`Privacy`: Select the visibility of this meeting. Options are :guilabel:`Public`,
+  :guilabel:`Private`, and :guilabel:`Only internal users`. :guilabel:`Public` allows for everyone
+  to see the meeting, :guilabel:`Private` allows only the attendees listed on the meeting to see the
+  meeting, and :guilabel:`Only internal users` allows everyone in the company to see the meeting.
+- :guilabel:`Show as`: Select from the drop-down whether the meeting time should appear as
+  :guilabel:`Busy` or :guilabel:`Available` in the attendees' schedules. Setting this field to
+  :guilabel:`Available` allows other meetings to be scheduled for the same time. It is recommended
+  to set this field to :guilabel:`Busy`, so no other meetings can be scheduled for the attendees at
+  the same time.
+
+.. image:: recruitment/new-event.png
+   :align: center
+   :alt: A new meeting card with all the details populated, and ready to save.
+
+Send meeting to attendees
+*************************
+
+Once changes have been entered and the meeting details are correct, the meeting can be sent to the
+attendees via email or text message.
+
+To send the meeting via email, click the :guilabel:`Email` button next to the list of attendees. A
+:guilabel:`Contact Attendees` pop-up appears. A pre-formatted email using the default
+:guilabel:`Calendar: Event Update` email template populates the email body field. The
+:guilabel:`Recipients` are by default the followers of the document (job application) as well as
+the :guilabel:`Recruiter`. Add the applicant's email address to the list to send the email to the
+applicant as well. Make any other desired changes to the email. If an attachment is needed, click
+the :guilabel:`Attach A File` button, navigate to the file, then click :guilabel:`Open`. Once the
+email is ready to be sent, click :guilabel:`Send`.
+
+.. image:: recruitment/email-event.png
+   :align: center
+   :alt: Enter the information to send the event via email.
+
+To send the meeting via text message, click the :guilabel:`SMS` button next to the list of
+attendees. A :guilabel:`Send SMS Text Message` pop-up appears. At the top, in a blue box,
+information appears regarding the message. The first line lists how many records (phone numbers) are
+selected. It is *not* reccommended to check the box to send to all the records. The second line lists
```

Comment:
Do we have a concrete reason why we don't want to send to all records? (Beyond the obvious spamming?) If so, it may be worth a "Warning" call out.

Reply:
Good idea!